### PR TITLE
Fix package name

### PR DESCRIPTION
**Pull request by charleskawczynski**
Hi @odunbar, @dburov190, and @agarbuno

We need to standardize the package name. Please check that these changes don't break any results/tests. If they don't, feel free to merge and we can push forward. Once this is configured, we can set up some automated CI, so that changes can be made more easily.

Note that I've also changed the tolerance of `1e-8` to `sqrt(eps(Float64))`

### CODE REVIEW

File: docs/src/index.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# CES.jl
+
+Stochastic Optimization, Learning, Uncertainty and Sampling
```
Comment (tapios) [inline docs/src/index.md]:
```suggestion
Calibrate, Emulate, Sample
```

File: src/neki.jl [hunk 2]
```diff
@@ -1,28 +1,28 @@
 """
-    neki_iter(prob::SolusProblem, θs, fθs)
+    neki_iter(prob::CESProblem, θs, fθs)
```
Comment (tapios) [inline src/neki.jl]:
```suggestion
    eki_iter(prob::CESProblem, θs, fθs)
```
Change `neki` -> `eki` throughout, including in file name.

File: src/neki.jl [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
 
-Peform an iteration of NEKI, returning a new set of proposed `θs`.
+Perform an iteration of NEKI, returning a new set of proposed `θs`.
```
Comment (tapios) [inline src/neki.jl]:
```suggestion
Perform an iteration of EKI, returning a new set of proposed `θs`.
```

File: src/neki.jl [hunk 4]
```diff
@@ -1,28 +1,28 @@
...
 
 See eqs. (5.4a-b) from https://arxiv.org/abs/1903.08866.
 """
-function neki_iter(prob::SolusProblem, θs, fθs)
+function neki_iter(prob::CESProblem, θs, fθs)
```
Comment (tapios) [inline src/neki.jl]:
```suggestion
function eki_iter(prob::CESProblem, θs, fθs)
```

File: src/neki.jl [hunk 5]
```diff
@@ -1,28 +1,28 @@
...
     covθ = cov(θs)
     meanθ = mean(θs)
-    
+
     m = mean(fθs)
 
     J = length(θs)
     CG = [dot(fθk - m, fθj - prob.obs, prob.space)/J for fθj in fθs, fθk in fθs]
 
     Δt = 1.0 / (norm(CG) + 1e-8)
```
Comment (tapios) [inline src/neki.jl]:
```suggestion
    Δt = 1.0 / (norm(CG) +eps)
```

File: src/neki.jl [hunk 6]
```diff
@@ -1,28 +1,28 @@
 """
-    neki_iter(prob::SolusProblem, θs, fθs)
+    neki_iter(prob::CESProblem, θs, fθs)
 
-Peform an iteration of NEKI, returning a new set of proposed `θs`.
+Perform an iteration of NEKI, returning a new set of proposed `θs`.
 
 See eqs. (5.4a-b) from https://arxiv.org/abs/1903.08866.
```
Comment (tapios) [inline src/neki.jl]: Update reference, @agarbuno?

File: test/neki.jl [hunk 7]
```diff
@@ -16,14 +16,14 @@ using LinearAlgebra, Distributions
...
-    prob = Solus.SolusProblem(prior, f, y_obs, Solus.CovarianceSpace(Γ))
+    prob = CES.CESProblem(prior, f, y_obs, CES.CovarianceSpace(Γ))
 
     J = 50
     θs = [rand(2) for i in 1:J]
 
     for i = 1:20
         fθs = map(f, θs)
-        θs = Solus.neki_iter(prob, θs, fθs)
+        θs = CES.neki_iter(prob, θs, fθs)
```
Comment (tapios) [inline test/neki.jl]:
```suggestion
        θs = CES.eks_iter(prob, θs, fθs)
```

File: src/neki.jl (hunk 5 above)
Comment (charleskawczynski) [reply src/neki.jl]: Since this may change the results, I'm going to leave this for a later PR and open an issue.

**Comment by ali-ramadhan** [comment]
Not sure if the plan is to register this as a Julia package but the name CES.jl might get rejected as it goes against naming guidelines 1 and 4: https://julialang.github.io/Pkg.jl/v1/creating-packages/#Package-naming-guidelines-1

1. Avoid jargon. In particular, avoid acronyms unless there is minimal possibility of confusion.
4. Err on the side of clarity, even if clarity seems long-winded to you.

**Comment by charleskawczynski** [comment]
Thanks @ali-ramadhan! Good point. We'll want to change the name, but I'm not sure what the final name should be. I'm probably not the person to decide on the name. This PR just makes the naming consistent because, before, `CES` and `Solus` were both used but in different places, and I couldn't configure any tests.

**Comment by charleskawczynski** [comment]
Is `CalibrateEmulateSample.jl` too verbose? @agarbuno @dburov190 @odunbar

**Comment by tapios** [comment]
‘CalibrateEmulateSample.JPL’ is good. You can also use ‘EnsembleKalmanSampler’ for ‘EKS’.

**Comment by tapios** [comment]
Please also change the 1e-8 to ‘sqrt(eps)’. Whatever small effect it has on results is not material. We should address such issues when we can. Github issues seem to linger for a long time before being acted upon.

**Comment by charleskawczynski** [comment]
Closes #13

File: examples/GPR/main.jl [hunk 8]
```diff
@@ -4,9 +4,9 @@ import NPZ
 import PyPlot
 const plt = PyPlot
 
-include("../../src/GPR.jl")
+include(joinpath("..","..","src","GPR.jl"))
 
-const gpr_data = NPZ.npzread("../../test/GPR/data/data_points.npy")
+const gpr_data = NPZ.npzread(joinpath("..","..","test","GPR","data","data_points.npy"))
```
Comment (dburov190) [inline examples/GPR/main.jl]: Maybe a minor point, but is this standard or something? don't see why the proposed way is clearer/better

Comment (charleskawczynski) [reply examples/GPR/main.jl]: It's more OS friendly. For poor windows users (like myself) this translates to `..\\..\\test\\GPR\\data\\data_points.npy`.

File: src/neki.jl (hunk 6 above)
Comment (charleskawczynski) [reply src/neki.jl]: Should we change this to `https://arxiv.org/pdf/2001.03689.pdf`?

File: examples/GPR/main.jl (hunk 8 above)
Comment (dburov190) [reply examples/GPR/main.jl]: oh, I see
funnily enough, I'm myself a windows user (at the time being, hopefully not for long) and this works for me, i.e. I wrote this code :D I guess, mingw makes the difference? anyway, I'm always in for cross-platform

**Comment by charleskawczynski** [comment]
> Seems like this does break a bit of my stuff (examples directory), hold off a little bit, I'll fix that soon and then we can merge.

Let me know if there's anything I can help with.

**Comment by charleskawczynski** [comment]
@dburov190 @odunbar any updates on merging this?

**Pull request merged**